### ISSUE TITLE
Question about link ST formatter

### ISSUE THREAD

**Issue by varloc2000** [mindrevolution/SirTrevor-for-Umbraco#24]
Hello!
[Here in your code](https://github.com/mindrevolution/SirTrevor-for-Umbraco/blob/master/src/App_Plugins/SirTrevor/formatters/umbraco.js#L51) you restore selection after linkPicker trigger their callback to correctly execute command. Have you test this in different browsers?

I'm just asking because i'm trying to create alike formatter but use jquery dialog as linkPicker form, and in my code restore selection in chrome browser don't work - selection does not have time to recover before executing the command or something....
But if I add debug point in my code in line with restoreSelection call - all work fine even in chrome.

Maybe you can help? Know something about that strange behavior?

**Comment by MarcStoecker** [comment]
I didn't experience any strange behaviour, honestly. I'm mostly developing on Chrome. :)

**Comment by varloc2000** [comment]
Ok. I figured out the cause of such oddities! 
jQuery ui dialog plugin I use for linkPicker - call jQuery .focus() on first field in dialog if present and it somehow broke process - restoreSelection than execCommand.

I added following html to prevent focusing...or if i add 'disabled' attr to url field like in your linkPicker all work fine too)

``` html
<input type="hidden" autofocus="autofocus" />
```

Anyway. Thanks for fast response)
